gradle-plugins/compose/src/main/kotlin/org/jetbrains/compose/ComposeCompilerCompatibility.kt

```diff
@@ -23,6 +23,7 @@ internal object ComposeCompilerCompatibility {
         "1.9.20" to "1.5.3",
         "1.9.21" to "1.5.4",
         "1.9.22" to "1.5.8.1",
+        "1.9.23" to "1.5.10",
```

Comment:
Also let's udpate here: https://github.com/JetBrains/compose-multiplatform/blob/shish/compiler/release/1.5.10-for-compose-1.6.1/gradle-plugins/gradle.properties#L15-L17

Reply:
done